### PR TITLE
Zims cleanup branch

### PR DESCRIPTION
**Pull request by zikalino**
@ZhijunZhao

### CODE REVIEW

File: all.js [hunk 1]
```diff
@@ -8,6 +8,17 @@ var simssh = require('simple-ssh');
 var scp2 = require('scp2')
 var biHelper = require('./biHelper.js');
 
+// [REVIEW] we should have only one uploadFiles, either uploadFiles, or uploadFilesViaScp
```
Comment (ZhijunZhao) [inline all.js:11]: Let's remove 'uploadFiles' and use 'uploadFilesViaScp' only, since the later support folder creation.

File: all.js [hunk 2]
```diff
@@ -8,6 +8,17 @@ var simssh = require('simple-ssh');
...
+// [REVIEW] sourceFileList + targetFileList doesn't seem to be elegant solution
```
Comment (ZhijunZhao) [inline all.js:12]: Yes, we can improve it if we don't need to support the copy-and-rename case.

File: all.js (hunk 1 above)
Comment (zikalino) [reply all.js:11]: yes, that's what i was thinking, uploadFiles is only used in raspberry pi c, and I can just switch to SCP as well

File: raspberrypi-c.js [hunk 3]
```diff
@@ -21,6 +23,18 @@ var COMPILER_FOLDER = all.getToolsFolder() + '/' + COMPILER_NAME + '/bin';
 //  all.gulpTaskBI(gulp, 'c', 'RaspberryPi', 'az-blink');
 //}
 
+// XXX - this function shall be replaced
+function azhRunLocalCmd(cmd, verbose, cb) {
```
Comment (ZhijunZhao) [inline raspberrypi-c.js:27]: > azhRunLocalCmd [](start = 9, length = 14)

Are you planning to change this in a separate PR?

Comment (zikalino) [reply raspberrypi-c.js:27]: yes, that was my plan, to discuss raspberry pi node changes with you and then change along with raspberry pi changes

**Review by ZhijunZhao — APPROVED** [review]

**Pull request merged**